pkg/controller/usersignup/usersignup_controller_test.go

```diff
@@ -1770,6 +1843,84 @@ func TestUserSignupDeactivatingWhenMURExists(t *testing.T) {
 	})
 }
 
+func TestUserSignupDeactivatingNotificationCreated(t *testing.T) {
+	userSignup := &v1alpha1.UserSignup{
+		ObjectMeta: NewUserSignupObjectMeta("", "edward.jones@redhat.com"),
+		Spec: v1alpha1.UserSignupSpec{
+			UserID:   "UserID089",
+			Username: "freja.johanssen@redhat.com",
+			States:   []v1alpha1.UserSignupState{"deactivating"},
+		},
+		Status: v1alpha1.UserSignupStatus{
+			Conditions: []v1alpha1.Condition{
+				{
+					Type:   v1alpha1.UserSignupComplete,
+					Status: v1.ConditionTrue,
+				},
+				{
+					Type:   v1alpha1.UserSignupApproved,
+					Status: v1.ConditionTrue,
+					Reason: "ApprovedAutomatically",
+				},
+			},
+			CompliantUsername: "freja-johanssen",
+		},
+	}
```

Comment:
you could use functions from https://github.com/codeready-toolchain/host-operator/blob/master/test/usersignup.go to simplify it